### ISSUE TITLE
[EN][Spell][Talent][Class] Ice Trap zone and Lock and Load proc

### ISSUE THREAD

**Issue by Shanrolol** [Atlantiss/BugTracker#8791]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

Good time of the day, everyone. I've found an issue with a Lock and Load proc (http://cata.openwow.com/spell=56343), being that if you land an Ice Trap (http://cata.openwow.com/spell=13809) on a susceptible target (no magic or slow/root immunity) and after the ICD of the proc is over (being roughly 10-12 seconds), however, the target is affected by an icy zone of a previous Ice Trap, you wont get the proc. So, to say it simplier, if you happen to stack two Ice Trap zones on one target, with all conditions for the proc cleared, you still wont get it. This happens to be the last issue there is to the mechanic, everything else is working properly.

**Comment by kadi2** [comment]
Hey, you have to follow bug tracker rules and prolly post some proofs.

**Comment by Shanrolol** [comment]
Sure, there you go.
![issue 1](https://cloud.githubusercontent.com/assets/16698440/21953444/5ec49d52-da61-11e6-8f64-02b681304163.jpg)
![issue 2](https://cloud.githubusercontent.com/assets/16698440/21953445/63cffcc4-da61-11e6-9b30-9ab1aac1cee2.jpg)

**Comment by TranceAccount** [comment]
You are supposed to follow the issue template which is inserted automatically by Github.

**Comment by Shanrolol** [comment]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: Consecutive Ice Traps (http://cata.openwow.com/spell=13809) dont proc Lock and Load (http://cata.openwow.com/spell=56343) if their fields stack on one target.
![issue 1](https://cloud.githubusercontent.com/assets/16698440/22033152/983029c8-dd11-11e6-9562-4ef4a5ca405c.jpg)
![issue 2](https://cloud.githubusercontent.com/assets/16698440/22033158/9af2dac0-dd11-11e6-9cc1-b30c92b6991b.jpg)



**Current behaviour**: If you use an Ice Trap on a completely susceptible target (no magic or slow/root immunity), that already has an icy field of a previous Ice Trap beneath its feet, you dont get a Lock and Load proc. Apparently, that happens because of the old icy field debuff overrides the new one on the target.

**Expected behaviour**: A new Ice Trap is supposed to simply create another icy field, next to the old one, and its debuff effect should override that of an older one, thus, granting you a Lock and Load proc properly.

**Steps to reproduce the problem**:

1. Go to the dummy location as a survival hunter and drop an Ice Trap beneath any dummy, and get a Lock and Load proc.
2. Wait for its cooldown to pass, and drop another Ice Trap on the same dummy, this second time you will get no Lock and Load proc. Notice that the new traps slow debuff will only be seen, after old one's is over.
3. Just for the interest, you can try dropping two Ice Traps on two different dummies, like I did, you are going to get two separate icy fields and two proper Lock and Load procs.

**Revision**: 5291 52595540b7ac 2017-01-17

**Comment by Batranelu** [comment]
Confirm , second trap launched isnt working if it's the same trap.

**Comment by Shanrolol** [comment]
Yes, it works with Freezing into Ice, or the other way around. Its the Ice trap to second Ice trap that doesnt work. The Lock and Load proc is triggered by the moment, when a target is being affected by a debuff of the trap. Freezing into second Freezing is simple, there is no zone to stay in to get the debuff, its just a single target cc effect, that just gets refreshed and you get the proc. As a fix for Ice into second Ice, they could just let the target, standing on two stacked icy fields to be affected by two separate debuffs, older one, nearing its end, and a freshly applied new one, giving a proper proc. It's not like the slow will become greater than 50% because of that, it would be like being affected by two wing clips from different hunters.
